app/_hub/imperva/imp-appsec-connector/_index.md

```diff
@@ -0,0 +1,285 @@
+    - name: queue_size
+      required: no
+      default: 1
+      datatype: number
+      encrypted: false
+      value_in_examples: 1
+      minimum_version: 3.0.0
+      maximum_version: 3.2.0
+      description: Number of API logs to keep in queue for retry. Default is 1 meaning no retry.
+    - name: flush_timeout
+      required: no
+      default: 2
+      datatype: number
+      encrypted: false
+      value_in_examples: 2
+      minimum_version: 3.0.0
+      maximum_version: 3.2.0
+      description: Number of seconds to wait before flushing the queue. 
+    - name: timeout
+      required: no
+      default: 6000000
+      datatype: number
+      encrypted: false
+      value_in_examples: 6000000
+      minimum_version: 3.0.0
+      maximum_version: 3.2.0
+      description: Number of milli-seconds to keep a single connection open to the destination. 
+  extra:
+    # This is for additional remarks about your configuration.
+
+
+###############################################################################
+# END YAML DATA
+# Beneath the next --- use Markdown (redcarpet flavor) and HTML formatting only.
+#
+# The remainder of this file is for free-form description, instruction, and
+# reference matter.
+# If you include headers, your headers MUST start at Level 2 (parsing to
+# h2 tag in HTML). Heading Level 2 is represented by ## notation
+# preceding the header text. Subsequent headings,
+# if you choose to use them, must be properly nested (eg. heading level 2 may
+# be followed by another heading level 2, or by heading level 3, but must NOT be
+# followed by heading level 4)
+###############################################################################
+# BEGIN MARKDOWN CONTENT
+---
+## How it works
+The plugin sends a copy of API call requests/responses to the Imperva API receiver. The receiver service destination address and port are specified as config parameters. Additional parameters are used to control how the API captures are sent. 
+
+### How to install
+
+> If you are using Kong's [Kubernetes Ingress Controller](https://github.com/Kong/kubernetes-ingress-controller), the installation is slightly different. Review the [docs for Kubernetes Ingress](https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/setting-up-custom-plugins/)
+
+The `.rock` file is a self-contained package that can be installed locally or from a remote server.
+
+If the LuaRocks utility is installed in your system (this is likely the case if you used one of the official installation packages), you can install the 'rock' in your LuaRocks tree (a directory in which LuaRocks installs Lua modules).
+
+### Install the Imperva Plugin
+
+```shell
+ luarocks install imp-appsec-connector
+```
+
+### Update your loaded plugins list
+In your `kong.conf`, append `imp-appsec-connector` to the `plugins` field (or `custom_plugins` if old version of Kong). Make sure the field is not commented out.
```

Comment:
```suggestion
In your `kong.conf`, append `imp-appsec-connector` to the `plugins` field. Make sure the field is not commented out.
```
Not compatible with older versions. I think `custom_plugins` might be something from pre-1.0.